packages/dev/s2-docs/pages/react-aria/releases/v1-11-0.mdx

```diff
@@ -0,0 +1,241 @@
+* Select/Picker
+  * Allow `selectedKey={null}` to clear value in `HiddenSelect` - [@chirokas](https://github.com/chirokas) - [PR](https://github.com/adobe/react-spectrum/pull/8330)
+  * Fix `HiddenSelect` causing page scrolling - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/8579)
+  * Fix keyboard selection in `Select` typeahead when options include spaces - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/8497)
+  * Prevent form submission when required `Select` has more than 300 options - [@lucasweng](https://github.com/lucasweng) - [PR](https://github.com/adobe/react-spectrum/pull/8280)
+* Tabs
+   * Fix wrapping with vertical `TabList` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/8602)
+  * Match id type in `Tab` and `TabPanel` - [@kidonng](https://github.com/kidonng) - [PR](https://github.com/adobe/react-spectrum/pull/8423)
+* Miscellaneous
+  * Check user agent brands - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/8458)
+  * Fix `Rect` intersection check so that users don't need to provide Virtualizer mocks for tests - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/8396)
+  * Fix `createLeafComponent` ref typing - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/8507)
+  * Add permission element to focusable elements list - [@gabrielchl](https://github.com/gabrielchl) - [PR](https://github.com/adobe/react-spectrum/pull/8356)
+  * Minimize unnecessary global events in `useHover` - [@qmhc](https://github.com/qmhc) - [PR](https://github.com/adobe/react-spectrum/pull/8483)
+  * Do not dispatch `loadMore` if hook is still in loading state  - [@jluyau](https://github.com/jluyau) = [PR](https://github.com/adobe/react-spectrum/pull/8571)
+
+## Docs
+* Remove duplicate "color" in `ColorWheelRenderProps` comment - [@GuiEpi](https://github.com/GuiEpi) - [PR](https://github.com/adobe/react-spectrum/pull/8465)
+* Fix `Calendar` example - [@chirokas](https://github.com/chirokas) - [PR](https://github.com/adobe/react-spectrum/pull/8529)
+* Use `lucide-react` for React Aria example's icons - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/8516)
+* Fix TanStack Router example - [@brmzkw](https://github.com/brmzkw) - [PR](https://github.com/adobe/react-spectrum/pull/8404)
+* Fix checkbox in dark mode in the Tailwind Starter - [@tomer-yechiel](https://github.com/tomer-yechiel) - [PR](https://github.com/adobe/react-spectrum/pull/8379)
+
+## Under Construction
+* Autocomplete
+  * Clear Autocomplete virtualFocus upon paste/undo/redo and other focus fixes - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/8438)
+* Tree:
```

Comment:
<img width="736" height="222" alt="Image" src="https://github.com/user-attachments/assets/48366d96-3b16-4202-8b42-a935874c0ecc" />

double spaced, can be follow-up